coralproject/talk
# issue #1022 - quiet logs on production mode

## What does this PR do?

removes console warnings and logs on production mode (issue #1022)

## How do I test this PR?

- build the app on production mode
- load client served by this build
- you shouldn't see any warnings except this one (This page does not include a canonical link tag. Talk has inferred this asset_url from the window object. Query params have been stripped, which may cause a single thread to be present across multiple pages.)
